### DISCUSSION TITLE
[Bug]: Moving from v6.10 to Dev breaks OpenAI Settings

### DISCUSSION THREAD

**Discussion by carydias** [danny-avila/LibreChat#2038] [Troubleshooting]
### What happened?

I moved from v6.10 (release) to the dev channel so that I could use Claude v3.

OpenAI models now do not work, and return the following error:
`Something went wrong. Here's the specific error message we encountered: Invalid JSON provided for openAI user values. Please provide them again.`

Rolling back to the Stable channel causes OpenAI to work again.

Is this a new breaking change?  How can I help debug it?  

### Steps to Reproduce

1. Change build channel in Dockerfile from FROM ghcr.io/danny-avila/librechat:latest to FROM ghcr.io/danny-avila/librechat-dev:latest
2. Create a new chat with the OpenAI models
3. Try and send a query

### Relevant log output

```shell
error: [handleAbortError] AI response error; aborting request: Invalid JSON provided for openAI user values. Please provide them again.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
This is an expected breaking change. The fix is to input your API key again, have you done this?

**Reply by carydias** [reply]
I have tried updating the API key - yes... 

I read through the source code but couldn't follow what it mean by "openAI user values" because I had it deployed on Huggingface and I assume I cannot see the appropriate config files.  Can I see them and debug it for all?
